Realm/RLMProperty.mm

```diff
@@ -722,6 +724,9 @@ - (NSString *)columnName {
 - (realm::Property)objectStoreCopy:(RLMSchema *)schema {
     realm::Property p;
     p.name = self.columnName.UTF8String;
+    if (self.columnName != _name) {
```

Comment:
I think `if (_columnName)` is clearer here. Comparing NSStrings by pointer is something that inherently looks very suspicious, even though it is functionally correct here.

Reply:
Thanks for the tip